packages/analytics/shippers/elastic_v3/server/src/server_shipper.ts

```diff
@@ -278,13 +273,13 @@ export class ElasticV3ServerShipper implements IShipper {
   }
 
   /**
-   * Returns a queue of events of up-to 10kB.
+   * Returns a queue of events of up-to 10kB. Or all events in the queue if it's a FLUSH action.
```

Comment:
I added this extra use case: FLUSH action to ensure we clear the queue.

We can leverage this in https://github.com/elastic/kibana/issues/140521 in a follow-up PR by attaching a callback to be called whenever the queue is emptied.